### PR TITLE
Adds SyntaxHighlighting page

### LINKED ISSUE #101: Add syntax highlighting setup to tooling
We should add docs about all the syntax highlighters for all the different editors to the website so people can find it!

### PR DESCRIPTION
**Pull request by SaraVieira**
https://styled-components-docs-cbndcopbbi.now.sh/docs/tooling#syntax-highlighting

closes #101

### CODE REVIEW

File: sections/tooling/syntax-highlighting.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+Also if you're looking for an awesome javascript syntax package you can never go wrong with [YAJS.vim](https://github.com/othree/yajs.vim).
+
+### WebStorm, IntelliJ IDEA, PhpStorm, PyCharm, and RubyMine
+
+The [\`webstorm-styled-components\`](https://github.com/styled-components/webstorm-styled-components) plugin adds code completion and highlighting for CSS properties and values in the template strings. And it also provides code completion and navigation for JavaScript symbols in the interpolations. You can install it from the IDE: go to \`Preferences | Plugins\` and search for \`Styled Components\`.
+
+### Other Editors
+
+We could use your help to get syntax highlighting support to other editors! If you want to start working on syntax highlighting for your editor, open an issue to let us know.
```
Comment (morajabi) [inline sections/tooling/syntax-highlighting.js]: Maybe add: `all this wouldn't happen without the community help, ...` or something similar to mention the current syntax highlighting solutions are made by the community.

File: sections/tooling/syntax-highlighting.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import md from 'components/md'
+
+const SyntaxHighlighting = () => md`
+  ## Syntax Highlighting
+
+  The one thing you lose when writing CSS in template literals is syntax highlighting. We're working hard on making proper syntax highlighting happening in all editors. We currently have support for Atom, Visual Studio Code, WebStorm, and soon Sublime Text.
```
Comment (morajabi) [inline sections/tooling/syntax-highlighting.js]: After reading the first sentence it feels like we are losing it now. Maybe slightly change it.

Comment (SaraVieira) [reply sections/tooling/syntax-highlighting.js]: Maybe put a past tense on it, good idea :)

File: sections/tooling/syntax-highlighting.js (hunk 1 above)
Comment (SaraVieira) [reply sections/tooling/syntax-highlighting.js]: Sure! I:)

**Comment by SaraVieira** [comment]
@morajabi changes made 👍

**Comment by mxstbr** [comment]
I don't know what's up with these test failures, /cc @philpl

**Comment by SaraVieira** [comment]
@mxstbr @philpl It passes on my pc :(

**Comment by philpl** [comment]
@SaraVieira There was an issue with a changed API in Puppeteer 0.13.0; We've fixed the integration tests, pinned the version and pushed a hotfix to master.

https://github.com/styled-components/styled-components-website/commit/c3e1499e58002103d3daf6d16375b6ac67bb1065

Can you please rebase your branch and run `yarn integration -u` to update the screenshots? This should fix the tests :wink:

**Comment by mxstbr** [comment]
Thank you so much for helping us improve styled-components! Based on our [Community Guidelines](https://github.com/styled-components/styled-components/blob/master/CONTRIBUTING.md) every person that has a PR of any kind merged is offered **an invitation to the styled-components organization**—that is you right now!

Should you accept, you'll get write access to the main repository. (and a fancy `styled-components` logo on your GitHub profile!) You'll be able to label and close issues, merge pull requests, create new branches, etc. We want you to help us out with those things, so don't be scared to do them! Make sure to read our [contribution and community guidelines](https://github.com/styled-components/styled-components/blob/master/CONTRIBUTING.md), which explains all of this in more detail. Of course, if you have any questions just let me know!

**Comment by mxstbr** [comment]
Thank you @SaraVieira!!

**Comment by SaraVieira** [comment]
Thank you peeps 😍

**Comment by morajabi** [comment]
Best repo ever :)

**Pull request merged**